src/_playground/App.scss

```diff
@@ -96,12 +95,28 @@ $library-color: #61dafb;
             }
         }
 
+        @media (max-width: $mediaMaxWidth) {
+            .frDocs-Sidebar__visibility {
```

Comment:
This class name isn't technically correct for BEM.  This would mean `visibility` is the element, whereas it should be modifier.  That said, I think the name itself is misleading.  The class name should be representative of _what_ it's being used for.  This class is something that gets added to two elements so you can make use of them as a CSS selector for the media queries.  Maybe `frDocs-Sidebar--isHidable`?